README.md

```diff
@@ -522,6 +522,43 @@ Usable properties for an unsubscribe `question` type are `title`, `content`, `bu
 The default `data_method` is `delete`. The `data_method` should typically not be specified as it should correspond with the `unsubscribe_url` that is supplied by the system when calling the questionnaire generator. Only when we call this private function with `send` to show a card on the mentor dashboard is when we override both the `unsubscribe_url` and the `data_method` but it's a bit of a hack.
 
 
+
+### Type: Dropdown
+Required and allowed options (minimal example and maximal example):
+
+```ruby
+[{
+  id: :v1,
+  type: :dropdown,
+  title: 'Waar hadden de belangrijkste gebeurtenissen mee te maken?',
+  options: ['hobby/sport', 'werk', 'vriendschap', 'romantische relatie', 'thuis']
+}, {
+  section_start: 'De hoofddoelen',
+  hidden: true,
+  id: :v2,
+  type: :dropdown,
+  title: 'Aan welke doelen heb je deze week gewerkt tijdens de begeleiding van deze student?',
+  label: 'RMC regio',
+  tooltip: 'some tooltip',
+  options: ['hobby/sport', 'werk', 'vriendschap', 'romantische relatie', 'thuis'],
+  section_end: true
+}]
+```
+
+The options array must contain of strings. Currently, there is no support for `shows_questions` or `hides_questions` triggers based on selected options in a dropdown.
```

Comment:
Ow, hier staat het. Willen we dat inderdaad niet?

Reply:
Meer omdat het lastig is te doen en we het nu niet nodig hebben. Misschien moeten we alles maar eens omzetten naar react dan zijn dit soort dingen vet makkelijk te doen.